src/utils/printResult.js

```diff
@@ -0,0 +1,15 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+/**
+ * 매 시도마다 실행 결과를 출력합니다.
+ * playersMap의 key(차 이름)가 전진한 만큼 (value) '-'를 출력합니다.
+ * @param {Map} playersMap
+ */
+export default function printResult(playersMap) {
+  playersMap.forEach((val, key) => {
+    let length = "";
+    for (let i = 0; i < val; i++) length += "-";
+    MissionUtils.Console.print(`${key} : ${length}`);
+  });
+  MissionUtils.Console.print("");
```

Comment:
```suggestion
  playersMap.forEach((val, key) => {
    const dash = "-".repeat(val)
    MissionUtils.Console.print(`${key} : ${dash}`);
  });
  MissionUtils.Console.print("\n");
```
forEach 안에 내부 반복문 부분을 `repeat` 함수로 간략하게 수정할 수 있을 것 같아요!

Reply:
와 `repeat`...를 몰랐네요 ㅎㅎ 뭔가 바닐라 JS가 더 어려운 느낌...ㅋㅋㅋ 감사합니당